frontend/src/ts/menu.ts

```
@@ -0,0 +1,49 @@
+/**
+ * @module menu-controls
+ * Module for handling kiosk menu interactions and image navigation
+ */
+
+import htmx from "htmx.org";
+
+let nextImageMenuButton: HTMLElement;
+let prevImageMenuButton: HTMLElement;
+
+let nextImageMenuButton: HTMLButtonElement;
```

Biome

[error] Shouldn't redeclare 'nextImageMenuButton'. Consider to delete it or rename it.

'nextImageMenuButton' is defined here:


(lint/suspicious/noRedeclare)